Mulligan.1.html

```diff
@@ -0,0 +1,174 @@
+<html>
+    <head>
+            <style>
+                 div.padded {  
+      padding-top: 50px;  
+      padding-right: 400px;  
+      padding-bottom: 0.25in;  
+      padding-left: 400px;  
+    }
+            #rcorners2 {
+            border-radius: 25px;
+            border: 10px solid; 
+            width: 200px;
+            height: 200px;
+            color: DodgerBlue;
+            float: left;
+            margin-right: 20px
+            
+                    }
+            #rcorners3 {
+            border-radius: 25px;
+            border: 10px solid; 
+            width: 200px;
+            height: 200px;
+            color: plum;
+            float: right;
+            margin-left: 20px
+                    }   
+            #rcorners4 {
+            border-radius: 25px;
+            border: 10px solid; 
+            width: 200px;
+            height: 200px;
+            color: orange;
+            float: left;
+            margin-right: 20px
+                    }        
+            #rcorners1 {
+            border-radius: 25px;
+            border: 10px solid; 
+            width: 200px;
+            height: 200px;
+            color:red;
+            float: right;
+            margin-left: 20px
+                    }          
+
+                    .main {
+             color: black;
+             font-size: 16px;
+             font-family: Helvetica;
+             
+             }
+
+            .name {
+             color: dodgerblue;
+             font-size: 30px;
+             font-family: Helvetica;
+             
+             }
+            .name2 {
+             color: plum;
+             font-size: 30px;
+             font-family: Helvetica;
+             }
+             .name3 {
+             color:orange;
+             font-size: 30px;
+             font-family: Helvetica;
+             }
+             .name4 {
+             color: red;
+             font-size: 30px;
+             font-family: Helvetica;
+             }
+
+             img.rounded-corners {
+border-radius: 15px;
+}
+
+             .image-container {
+  display: flex;
+  justify-content: center;
+}
+             
+        
+            
+            </style>
+    </head>
+        <body>
+                
+                <!-- get the header image to center on the page -->
+                <div class="image-container"><img src="TMNTHead.png" alt="TMNTHead" style="width:1400px;height:832px;"></div>
+            
+            <div class="padded">
+            <!-- see if you can get this a little bit narrower so that the text wraps to more lines to match the design a little closer -->
+            
+            <p style="font-family:Helvetica;">
+
+            <div id="rcorners2"><img src="leonardo.gif" alt="Leonardo" class="rounded-corners" style="float:left;width:200px;height:200px;">
+            </div>
+
+            <div style="padding-bottom:100px;"><span class="name"><b>Leonardo</b></span>
```

Comment:
This span should be a heading  element"`<h2>`"so it follows a semantic HTML approach for improved readability, accessibility, and ADA requirements.

Read: http://accessiblehtmlheadings.com/
Read: https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_HTML_sections_and_outlines